README.md

```diff
@@ -80,6 +80,7 @@ Recommended provisioner versions with Kubernetes versions
 
 | Provisioner version | K8s version   | Reason                    |
 | ------------------- | ------------- | ------------------------- |
+| [2.3.1][5]          | 1.12          |                           |
 | [2.3.0][4]          | 1.12          | fs on block support       |
```

Comment:
Maybe just replace 2.3.0 with 2.3.1? It's the same minor release train.

Reply:
done